### PR TITLE
Correct Description Error for Maximum Gas Amount

### PR DESCRIPTION
**Pull request by junkil-park**
### Description

This PR fix the error in the description about the max gas amount, and correctly states that it is in units, not in Octas.

### Checklist

- Do all Lints pass?
  - [] Have you ran `pnpm spellcheck`?
  - [] Have you ran `pnpm fmt`?
  - [] Have you ran `pnpm lint`?

Some of these checks seem to be already broken.

### CODE REVIEW

**Review by 0xaptosj — APPROVED** [review]

File: apps/docusaurus/docs/concepts/txns-states.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ A signed transaction on the blockchain contains the following information:
...
 - [**Gas unit price**](../reference/glossary.md#gas-unit-price): The amount the sender is willing to pay per unit of gas, to execute the transaction. This is represented as Octa or units of 10<sup>-8</sup> utility tokens.
-- [**Maximum gas amount**](../reference/glossary.md#maximum-gas-amount): The maximum gas amount in Aptos utility tokens the sender is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented as Octas or units of 10<sup>-8</sup> Aptos utility tokens.
+- [**Maximum gas amount**](../reference/glossary.md#maximum-gas-amount): The maximum gas amount in Aptos utility tokens the sender is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented in gas units, not Octas (i.e., 10<sup>-8</sup> Aptos utility tokens).
```
Comment (vgao1996) [inline apps/docusaurus/docs/concepts/txns-states.md:52]: Hmm... looks more more fixes are needed here

> The maximum gas amount in Aptos utility tokens

The maximum amount is measured in gas units

Might also help to clarify that the final fee is gas used * gas unit price

Comment (junkil-park) [reply apps/docusaurus/docs/concepts/txns-states.md:52]: I added the clarification.

File: apps/docusaurus/docs/concepts/txns-states.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ A signed transaction on the blockchain contains the following information:
...
-- [**Maximum gas amount**](../reference/glossary.md#maximum-gas-amount): The maximum gas amount in Aptos utility tokens the sender is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented as Octas or units of 10<sup>-8</sup> Aptos utility tokens.
+- [**Maximum gas amount**](../reference/glossary.md#maximum-gas-amount): The maximum gas amount in Aptos utility tokens the sender is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented in gas units, not Octas (i.e., 10<sup>-8</sup> Aptos utility tokens). The final gas charged is equal to the gas price multiplied by units of gas required to process this transaction. The required units of gas should not exceed the maximum gas amount for the transaction to be successfully executed.
```
Comment (vgao1996) [inline apps/docusaurus/docs/concepts/txns-states.md:52]: > The maximum gas amount **in Aptos utility tokens** the sender is willing to pay for this transaction.

Can you also rephrase this?

Comment (junkil-park) [reply apps/docusaurus/docs/concepts/txns-states.md]: Good catch! I fixed it as well.

**Review by vgao1996 — APPROVED** [review]

**Pull request merged**